meson.build

```diff
@@ -146,6 +146,9 @@ add_project_arguments('-DFWUPD_DISABLE_DEPRECATED', language : 'c')
 add_project_arguments('-D_BSD_SOURCE', language : 'c')
 add_project_arguments('-D_XOPEN_SOURCE=700', language : 'c')
 
+# needed for memfd_create()
+add_project_arguments('-D_GNU_SOURCE', language : 'c')
```

Comment:
Rather than `_BSD_SOURCE` and `_GNU_SOURCE`, can you just set `_DEFAULT_SOURCE`?

Reply:
No, it really needs to be `_GNU_SOURCE`